src/renderer/themes.css

```diff
@@ -550,6 +598,67 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --primary-color-active: #8d98e4;
 }
 
+.mainSolarizedYellow,
```

Comment:
You have a difference in how you group the main solarized colors and sec solarized colors, but they should be using the same split. #fdf6e3 for the main "dark" text color and `.secSolarizedBlue` & `.secSolarizedCyan`'s `#000`. These light and dark groupings need   `url("../../_icons/iconSolarizedLightSmall.svg")` & `url("../../_icons/textSolarizedLightSmall.svg")`, and `url("../../_icons/iconSolarizedDarkSmall.svg")` &  `url("../../_icons/textSolarizedDarkSmall.svg")`, respectively.

Reply:
I don't really understand what you mean. I tried to understand what you meant and came up with this:
```.mainSolarizedYellow,
.mainSolarizedOrange,
.mainSolarizedRed,
.mainSolarizedMagenta,
.mainSolarizedViolet,
.mainSolarizedGreen {
  --text-with-main-color: #000;
  --logo-icon-bar-color: url("../../_icons/iconDraculaDarkSmall.svg");
  --logo-text-bar-color: url("../../_icons/textDraculaDarkSmall.svg");
}

.mainSolarizedBlue,
.mainSolarizedCyan {
  --text-with-main-color: #fdf6e3;
  --logo-icon-bar-color: url("../../_icons/iconDraculaLightSmall.svg");
  --logo-text-bar-color: url("../../_icons/textDraculaLightSmall.svg");
}
```

It this the change you mean (grouping sec and main in the same way)?

Reply:
Almost, just with the bodies switched:
```
.mainSolarizedOrange,
.mainSolarizedRed,
.mainSolarizedMagenta,
.mainSolarizedViolet,
.mainSolarizedGreen {
  --text-with-main-color: #000;
  --logo-icon-bar-color: url("../../_icons/iconDraculaDarkSmall.svg");
  --logo-text-bar-color: url("../../_icons/textDraculaDarkSmall.svg");
}

.mainSolarizedBlue,
.mainSolarizedCyan {
  --text-with-main-color: #fdf6e3;
  --logo-icon-bar-color: url("../../_icons/iconDraculaLightSmall.svg");
  --logo-text-bar-color: url("../../_icons/textDraculaLightSmall.svg");
}
```

To validate, enable "Match Top Bar with Main Color" in theme settings, set the primary and secondary colors to the same color, and confirm that the `Data Settings` colors line up with the top bar colors.

Reply:
This has been implemented in https://github.com/FreeTubeApp/FreeTube/pull/5066/commits/7db56b47e09a60f1f7870a03d3fb4d1c8292b36c